pkg/scaffold/template.go

```diff
@@ -76,3 +42,66 @@ func SaveTemplate(fs *afero.Afero, path string, template []byte) error {
 
 	return err
 }
+
+// goTemplateToBytes converts a Go template plus provided data to a string
+func goTemplateToBytes(templateString string, data interface{}) ([]byte, error) {
+	tmpl, err := template.New("t").Parse(templateString)
+	if err != nil {
+		return nil, err
+	}
+
+	tmplBuffer := new(bytes.Buffer)
+	err = tmpl.Execute(tmplBuffer, data)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return tmplBuffer.Bytes(), nil
+}
+
+const okctlAppTemplate = `# A name that identifies your app
+name: my-app
+# An URI for your app Docker image
+image: docker.pkg.github.com/my-org/my-repo/my-package
```

Comment:
We should be creating ECR repositories on behalf of the user at some point in time

Reply:
We'll have at look at this in a future PR.